src/lib/ng-package/entry-point/write-package.transform.ts

```diff
@@ -11,9 +11,9 @@ import { EntryPointNode, isEntryPointInProgress, isPackage, PackageNode, fileUrl
 import { Node } from '../../graph/node';
 
 export const writePackageTransform: Transform = transformFromPromise(async graph => {
-  const entryPoint = graph.find(isEntryPointInProgress()) as EntryPointNode;
+  const entryPoint: EntryPointNode = graph.find(isEntryPointInProgress());
   const ngEntryPoint: NgEntryPoint = entryPoint.data.entryPoint;
-  const ngPackageNode = graph.find(isPackage) as PackageNode;
+  const ngPackageNode: PackageNode = graph.find(isPackage);
```

Comment:
@alan-agius4, do you prefer using inferred types in all situations, like:
```
const ngPackageNode = graph.find(isPackage);
```
instead of
```
const ngPackageNode: PackageNode = graph.find(isPackage);
```
This will work now with any type casting, but I've added the types just to inform the user that now we have a different type. When dealing with type guards I usually add the type to the returned value. But let me know if you want to remove them; we should strive for consistency.